test/extended/cluster/cl.go

```diff
@@ -179,8 +180,8 @@ var _ = g.Describe("[Feature:Performance][Serial][Slow] Load cluster", func() {
 		}
 
 		// Calculate and log test duration
-		metrics := []Metrics{NewTestDuration("cluster-loader-test", testStartTime, time.Since(testStartTime))}
-		err := LogMetrics(metrics)
+		m := []metrics.Metrics{metrics.NewTestDuration("cluster-loader-test", testStartTime, time.Since(testStartTime))}
+		err = metrics.LogMetrics(m)
```

Comment:
Not sure how did this build.  I had to revert back to `"err := metrics.LogMetrics(m)"` to compile this.